### ISSUE TITLE
Support for NodeJS v18.12.0 LTS

### ISSUE THREAD

**Issue by ebaauw** [oznu/homebridge-syno-spk#105]
Upgrading to NodeJS v18.12.0, now that has become the active LTS version.  Works on my mac and Raspberry Pi, but on Synology I get:
```
homebridge@ds2:/volume1/homebridge $ hb-service update-node
ℹ Updating Node.js from v16.17.1 to v18.12.0...
✖ Cannot update Node.js on your system. Synology DSM 7 does not currently support Node.js 18 or later.
```

Is this is limitation of the Homerbidge package or of NodeJS/Synology themselves?  Anything I can do?

**Comment by kellery82** [comment]
Just tried to update my node.js as well in my synology and i got the same message.

**Comment by oznu** [comment]
This is a limitation of the DSM 7 operating system. It does not run a new enough version of glibc.

**Comment by kellery82** [comment]
> This is a limitation of the DSM 7 operating system. It does not run a new enough version of glibc.

Thank you for the reply :)

**Comment by ebaauw** [comment]
So nothing we can do then, but wait for Synology.

**Comment by Gummibando** [comment]
Might be resolved in DSM 7.2, which contains glibc 2.36

**Comment by primovist** [comment]
#120
